src/partials/modal-window.html

```diff
@@ -0,0 +1,96 @@
+<div class="backdrop modal__backdrop is-hidden" data-modal>
+  <div class="modal container">
+    <button class="modal__close-btn" data-modal-close>
+      <svg class="modal__close-icon" width="14" height="14">
+        <use href="./images/svg/icons.svg#icon-close"></use>
+      </svg>
+    </button>
+    <form
+      class="callback container"
+      name="callback-form"
+      method="GET"
+      autocomplete="on"
+    >
+      <b class="callback__title"
+        >Please leave your details and we'll call you back to finalize your
+        order</b
+      >
+      <label class="input-element" for="modal-name">Your name</label>
+      <div class="input-element__field-wrapper">
+        <input
+          class="input-element__entry-field"
+          name="username"
+          type="text"
+          id="modal-name"
+          minlength="2"
+          maxlength="20"
+          placeholder=" "
+          required
+        />
+        <svg class="input-element__icon">
+          <use href="./images/svg/icons.svg#icon-person"></use>
+        </svg>
+      </div>
+      <label class="input-element" for="modal-tel">Phone</label>
+      <div class="input-element__field-wrapper">
+        <input
+          class="input-element__entry-field"
+          name="user-phone"
+          type="tel"
+          id="modal-tel"
+          pattern="[0-9]{3}-[0-9]{3}-[0-9]{2}-[0-9]{2}"
+          placeholder=" "
+          title="093-111-22-33"
+          required
+        />
+        <svg class="input-element__icon">
+          <use href="./images/svg/icons.svg#icon-phone"></use>
+        </svg>
+      </div>
+      <label class="input-element" for="modal-email">E-mail</label>
+      <div class="input-element__field-wrapper">
+        <input
+          class="input-element__entry-field"
+          name="user-email"
+          type="email"
+          id="modal-email"
+          placeholder=" "
+          required
+        />
+        <svg class="input-element__icon">
+          <use href="./images/svg/icons.svg#icon-email"></use>
+        </svg>
+      </div>
+      <label class="input-element" for="modal-comment">Purchare</label>
+      <textarea
+        class="input-element__coment-field"
```

Comment:
```suggestion
        class="input-element__comment-field"
```
Не принципиально для отображения, просто пепечатка. В src/sass/components/form-components/_input-elemet.scss тот же класс.